src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/commands.py

```diff
@@ -174,6 +174,10 @@ def load_command_table(self, _):  # pylint: disable=too-many-statements
         g.command('list', 'acr_task_list')
         g.command('delete', 'acr_task_delete')
         g.command('update', 'acr_task_update')
+        g.command('credential add', 'acr_task_credential_add')
+        g.command('credential update', 'acr_task_credential_add')
```

Comment:
I wonder if we need this update at all. If we have an update, should we block adding an existing credential, since it will `update` it?

Reply:
defer to @northtyphoon

Reply:
add new credential
update existing credential

what is current behavior? AddOrUpdate?